runtime/compiler/env/j9methodServer.cpp

```diff
@@ -2559,3 +2557,19 @@ TR_ResolvedRelocatableJ9JITaaSServerMethod::getAttributesCache(bool isStatic, bo
       }
    return attributesCache;
    }
+
+bool
+TR_ResolvedRelocatableJ9JITaaSServerMethod::validateMethodFieldAttributes(const TR_J9MethodFieldAttributes &attributes, bool isStatic, int32_t cpIndex, bool isStore, bool needAOTValidation)
+   {
+   // Validation should not be done against attributes cached per-resolved method
+   if (attributes.isUnresolvedInCP()) 
+      return true;
+   if (!isStatic)
+      _stream->write(JITServer::MessageType::ResolvedRelocatableMethod_fieldAttributes, getRemoteMirror(), cpIndex, isStore, needAOTValidation);
+   else
+      _stream->write(JITServer::MessageType::ResolvedRelocatableMethod_staticAttributes, getRemoteMirror(), cpIndex, isStore, needAOTValidation);
+   auto recv = _stream->read<TR_J9MethodFieldAttributes>();
+   auto clientAttributes = std::get<0>(recv);
+   bool equal = attributes == clientAttributes;
```

Comment:
Let's add parenthesis to make it clearer: `bool equal = (attributes == clientAttributes);` 
There is a similar construct in `TR_ResolvedJ9JITaaSServerMethod::validateMethodFieldAttributes` so that one needs adjustment as well.